editor/js/Sidebar.Project.Video.js

```diff
@@ -70,73 +70,108 @@ function SidebarProjectVideo( editor ) {
 
 		//
 
-		const { createFFmpeg, fetchFile } = FFmpeg; // eslint-disable-line no-undef
-		const ffmpeg = createFFmpeg( { log: true } );
+		const width = videoWidth.getValue();
+		const height = videoHeight.getValue();
 
-		await ffmpeg.load();
+		const duration = videoDuration.getValue();
+		const fps = videoFPS.getValue();
+		const frames = duration * fps;
+		const frameDuration = Math.floor( 1_000_000 / fps );
 
-		ffmpeg.setProgress( ( { ratio } ) => {
+		//
 
-			progress.setValue( ( ratio * 0.5 ) + 0.5 );
+		const mp4 = MP4Box.createFile();
+		let trackID = null;
+		let frame = 0;
 
-		} );
+		const encoder = new VideoEncoder( {
 
-		const fps = videoFPS.getValue();
-		const duration = videoDuration.getValue();
-		const frames = duration * fps;
+			output: function ( chunk, config ) {
 
-		let currentTime = 0;
+				if ( trackID === null ) {
 
-		for ( let i = 0; i < frames; i ++ ) {
+					trackID = mp4.addTrack( {
+						timescale: 1_000 * 1_000,
+						width: width,
+						height: height,
+						nb_samples: frames,
+						media_duration: duration * 1_000 * 1_000,
```

Comment:
I think numeric separators like in `1_000` aren't supported very much yet...

Reply:
Pretty good suuport actually: https://caniuse.com/mdn-javascript_grammar_numeric_separators